capstone/src/main/webapp/youtubeScript.js

```diff
@@ -264,9 +278,42 @@ function showCategories() {
     radiobox.onclick = function() {
       enableTextInput(this);   
     }
-    const container = document.getElementById('category-container');
-    container.appendChild(radiobox);
-    container.appendChild(label);
-    container.appendChild(document.createTextNode (" "));
+    const categoryContainer = document.getElementById('category-container');
+    categoryContainer.appendChild(radiobox);
+    categoryContainer.appendChild(label);
+    categoryContainer.appendChild(document.createTextNode(" "));
+  }
+}
+
+/** Converts perspective results to knoop scale then to mohs*/
+function perspectiveToxicityScale(attributeAverages) {
+  const knoopScale = [1, 32, 135, 163, 430, 560, 820, 1340, 1800, 7000];
```

Comment:
nit: can you add a comment about where these values come from?

Reply:
addressed in "add comment to show knoopScale source"